azure-spring-boot-parent/pom.xml

```diff
@@ -80,6 +80,16 @@
                 <artifactId>json</artifactId>
                 <version>20171018</version>
             </dependency>
+            <dependency>
+                <groupId>commons-logging</groupId>
+                <artifactId>commons-logging</artifactId>
+                <version>1.2</version>
```

Comment:
pls use properties for versions

Reply:
Because all the versions in parent is written with explicit version instead of property. So just added as explicit version.

Reply:
pls help to update it as properties, when 2.0 release, i add same comments in some already merged pr

Reply:
Moved version numbers to properties.